ramls/loan.json

```
@@ -12,6 +12,38 @@
       "description": "ID of the patron the item was lent to. Required for open loans, not required for closed loans (for anonymization).",
       "type": "string"
     },
+    "borrower": {
+      "description": "Copy of some requesting patron metadata (used for searching and sorting), will be taken from the user referred to by the requesterId",
```

Please change this description to reflect the usage in the loan.

It is the borrower of the item, rather than the requester. And these properties aren't stored copies (which they are for requests).
```suggestion
      "description": "Additional information about the borrower of the item, taken from the user referred to by the userId",
```

Does that make sense?

---

Sure